src/styles/product-card.scss

```diff
@@ -0,0 +1,104 @@
+.card {
+  width: $card-width;
+  box-sizing: border-box;
+  background-color: $background-color;
+  padding: 32px 16px 16px;
+
+  @include display-items(flex, column, center, center);
+  @include border-radius;
+  @include border(1px, solid, $border-color);
+
+  transition: transform 300ms;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+    display: block;
+    object-fit: cover;
+    object-position: center;
+  }
+
+  &__title {
+    width: 100%;
+    margin-bottom: 4px;
+    transition: all 300ms;
+
+    @include text-values(normal, 500, 12px, 18px, $primary-text-color, left);
```

Comment:
Ensure that the variable `$primary-text-color` is defined in your SCSS files. If it's not defined, it will cause an error during compilation.